docs/how-tos/hello-world-module.md

```diff
@@ -0,0 +1,509 @@
+---
+title: "Create a Hello World module"
+linkTitle: "Hello World module"
+type: "docs"
+weight: 24
+images: ["/registry/module-puzzle-piece.svg"]
+icon: true
+tags: ["modular resources", "components", "services", "registry"]
+description: "Get started writing your own modular resources by creating a Hello World module."
+languages: ["python", "go"]
+viamresources: ["components"]
+platformarea: ["registry"]
+level: "Beginner"
+date: "2024-10-22"
+# updated: ""  # When the tutorial was last entirely checked
+cost: "0"
+---
+
+This guide will walk you through creating two simple {{< glossary_tooltip term_id="modular-resource" text="modular resources" >}} that respond to API calls by returning a configured image, and a random number.
+By the end of this guide, you will be able to create your own [modular resources](/registry/modular-resources/) and package them into {{< glossary_tooltip term_id="module" text="modules" >}} so you can use them on your machines.
```

Comment:
I do want the link to go. I think in this case we've just added the same amount of information people would get from that page for this task. If they're here they don't need to look at configuration info or other available modular resources.

```suggestion
By the end of this guide, you will be able to create your own modular resources and package them into {{< glossary_tooltip term_id="module" text="modules" >}} so you can use them on your machines.
```